smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -65,7 +65,7 @@ contract MixinKeys is
   // the transfer of a key to another address where their key can be transferred
   // Note: the approver may actually NOT have a key... and there can only
   // be a single approved address
-  mapping (uint => address) private approved;
+  mapping (uint => address) internal approved;
```

Comment:
Any reason for the change? (curious...)

Reply:
this allows to access it from another mixin (private limits the scope inside the file itself)